File: launchdarkly/python/src/PatchFeatureFlagExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    patch_1 = models.PatchOperation(
        op="replace",
        path="/description",
    )

    patch = [
        patch_1,
    ]

    patch_with_comment = models.PatchWithComment(
        patch=patch,
    )

    try:
        response = api.FeatureFlagsApi(api_client).patch_feature_flag(
            project_key="projectKey_string",
            feature_flag_key="featureFlagKey_string",
            patch_with_comment=patch_with_comment,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling FeatureFlagsApi#patch_feature_flag: %s\n" % e)

File: launchdarkly/ruby/src/PatchFeatureFlagExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

patch_1 = LaunchDarklyClient::PatchOperation.new
patch_1.op = "replace"
patch_1.path = "/description"

patch = [
    patch_1,
]

patch_with_comment = LaunchDarklyClient::PatchWithComment.new
patch_with_comment.patch = patch

begin
    response = LaunchDarklyClient::FeatureFlagsApi.new.patch_feature_flag(
        "projectKey_string", # project_key
        "featureFlagKey_string", # feature_flag_key
        patch_with_comment,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling FeatureFlagsApi#patch_feature_flag: #{e}"
end
